### ISSUE TITLE
[Feature] Check if prefixed property exist 

### ISSUE THREAD

**Issue by evilebottnawi** [anandthakker/doiuse#27]
Add options for this feature and output (or not output) lower version browser support, based on browsers options. What do you think about?

**Comment by lorenz068** [comment]
Yes this is a good idea! :+1: Need this feature!

**Comment by anandthakker** [comment]
Agreed, prefixed property handling is something that needs work here.  See also #15 and #28

**Comment by dumblob** [comment]
Duplicate of https://github.com/anandthakker/doiuse/issues/91 ?
